### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by Homa3030**

### CODE REVIEW

File: app_python/PYTHON.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Pros:
+- Easy to understand development
+- It is very flexible and easy
+- Easy to test
+
+### Cons:
+- Timing in case of scaling
+- Using modules increases the security risk
+
+## References
```
Comment (afonyaa) [inline app_python/PYTHON.md:23]: It is great to pointing references for futher reading!

**Comment by afonyaa** [comment]
Good job, with concrete and needful description! I reccommend you to explicitly specify (host, port, may be in constants), where is your server are supposed to running

**Comment by none-word** [comment]
Good job!

**Comment by Maxkoz777** [comment]
Well done!

File: app_python/app.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import pytz
+from datetime import datetime, timezone
+from flask import Flask
+
+app = Flask(__name__)
+
```
Comment (AKurmazov) [inline app_python/app.py:7]: Nit: According to PEP8, "two blank lines are expected between functions and classes"
```suggestion


```

File: app_python/app.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import pytz
+from datetime import datetime, timezone
+from flask import Flask
```
Comment (AKurmazov) [inline app_python/app.py]: Nit: According to PEP8, imports should be sorted by the locality, and their alphabetic order
```suggestion
from datetime import datetime, timezone

import pytz
from flask import Flask
```

**Pull request closed without merging**